src/main/java/tc/oc/pgm/rotation/MapPoll.java

```diff
@@ -116,7 +129,9 @@ private Component getMapChatComponent(MatchPlayer viewer, MapInfo map) {
         new PersonalizedText(" ").bold(!voted), // Fix 1px symbol diff
         new PersonalizedText("" + countVotes(votes.get(map)), ChatColor.YELLOW),
         new PersonalizedText("] "),
-        map.getStyledName(NameStyle.FANCY));
+        map.getStyledMapName(
+            winner ? MapNameStyle.HIGHLIGHT_WITH_AUTHORS : MapNameStyle.COLOR_WITH_AUTHORS,
+            winner ? ChatColor.GREEN : ChatColor.GOLD));
```

Comment:
Why not just append a `.underlined(winner)` (or bold)?

Reply:
Originally I tried just modifying the map name as you suggested, however what I found was since the author list is included they are bolded/underlined as well. I feel that having just the map name highlighted is better visually. Also adjusting the color makes it more prominent too.